general/device.hpp

```diff
@@ -283,9 +283,9 @@ inline T *ReadWrite(Memory<T> &mem, int size, bool on_dev = true)
    }
 }
 
-/** @brief Shortcut to ReadWrite(const Memory<T> &mem, int size, false) */
+/** @brief Shortcut to ReadWrite(Memory<T> &mem, int size, false) */
 template <typename T>
-inline const T *HostReadWrite(const Memory<T> &mem, int size)
+inline T *HostReadWrite(Memory<T> &mem, int size)
```

Comment:
Shouldn't we do this too line 264 for the HostWrite?

Reply:
`HostWrite` is fixed in #937.

The fix in this PR could have been made there as well.